grpc/cmd/server/main.go

```diff
@@ -46,10 +61,11 @@ func runGRPC(lis net.Listener) {
 	pb.RegisterSimpleServerServer(server, NewServer())
 
 	log.Printf("gRPC Listening on %s\n", lis.Addr().String())
-	server.Serve(lis)
```

Comment:
I noticed that you are not catching the error that might be returned here so I decided to just return it from the function so that it can be caught from the **log.Fatal** in the main.